docs/xml_schemas.rst

```diff
@@ -0,0 +1,104 @@
+Configuration files
+####################
+
+Discovery-server operation is managed from a XML config file that follows the **discovery-server.xsd** schema located in
+**[SOURCES]**/discovery-server/resources/xsd (this is an extension of the fast-RTPS xml schema). The discovery-server
+main goals is:
+
+-   Simplify the configuration of fast-RTPS servers. Using a fast-RTPS participant profile for each server is tiresome,
+    given the large number of boilerplate code to move around. New XML syntax extensions are introduced to ease this task.
+
+-   Provide a flexible testing tool for the client-server discovery implementation. Testing the discovery involves
+    creating large number of participants, publishers, subscribers that use specific topics and types (static or dynamic
+    ones) over different transports. Besides all these entities may appear or disappear at different times, and we need to
+    be able to check the discovery status (collective participant knowledge) at any of these times.
+
+The outermost XML tag is **DS**. It admits an optional boolean attribute called **user_shutdown** that defaults to
+*true*. By default the discover-server binary runs indefinitely until the user decides to shutdown. This default
+behavior is suitable for practical applications but not for testing. Test XML files use :code:`user_shutdown="false"`,
+which grants that the discovery server is closed as soon as the test is fulfilled. The **DS** tag can contain the
+following tags:
+
++   **profiles** is plainly the fast-RTPS profiles. We can use them to fine tune the server operation. The associated
```

Comment:
Use tables to explain XML tags. Take a look into Fast-RTPS documentation for examples.